test/suites/dev/moonbase/test-randomness/test-randomness-babe-lottery2.ts

```diff
@@ -47,7 +47,7 @@ describeSuite({
         args: [0],
         account: BALTATHAR_ADDRESS,
       });
-      expect(estimatedGas).toMatchInlineSnapshot(`153999n`);
```

Comment:
The purpose of `toMatchInlineSnapshot` is to use a vitest mechanism to update the values automatically.
https://vitest.dev/api/expect.html#tomatchinlinesnapshot

Reply:
The problem with these snapshots was that they are taken outside of a test context:

> Error: 'toMatchInlineSnapshot' cannot be used without test context

I am not sure if this error is raised due to a TS dependency upgraded. 
On a related question, why don't we pin TS dependencies within the `package.json` file?

Reply:
But the snapshots are inside a context.

About the dependencies, I also don't like to not have them pinned. But the reason can be found here: https://github.com/moonbeam-foundation/moonbeam/pull/3091#issuecomment-2532496450

Reply:
I was getting these errors, but I guess it might be some local config problem. 👍